terraform/stack/dynamo.tf

```diff
@@ -15,9 +15,50 @@ module "replicator_lock_table" {
 locals {
   versioner_versions_table_name  = "${aws_dynamodb_table.versioner_versions_table.name}"
   versioner_versions_table_index = "ingestId_index"
+
+  replicas_table_name = "${aws_dynamodb_table.replicas_table.name}"
+}
+
+# TODO: Move this into the 'critical' part
+
+# Replicas
+
+resource "aws_dynamodb_table" "replicas_table" {
+  name      = "${var.namespace}_replicas_table"
+  hash_key  = "id"
+  range_key = "version"
+
+  billing_mode = "PAY_PER_REQUEST"
+
+  attribute {
+    name = "id"
+    type = "S"
+  }
+
+  attribute {
+    name = "version"
+    type = "N"
+  }
 }
 
-# TODO: MOve this into the 'critical' part
+data "aws_iam_policy_document" "replicas_table_readwrite" {
+  statement {
+    actions = [
+      "dynamodb:DeleteItem",
```

Comment:
Why do we need to grant DeleteItem privileges? Or UpdateItem? I would think the VersionedStore just needs Get, Query and Put.